tools/update-punycode.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
checking other files, I see `#!/bin/bash`, `#!/bin/sh` and `#!/usr/bin/env bash`. Not consistent.